ote/resources/public/language/en.edn

```diff
@@ -690,7 +690,9 @@
 
  :pre-notice-list-page
  {:header-pre-notice-list "Omat 60pv ilmoitukset"
+  :header-authority-pre-notice-list "Säännöllisen henkilöliikenteen muutosilmoitukset"
```

Comment:
I prefer own section for every page.
And I know that many of these texts are the same, but still. It might be easier to handle when texts are separated to different sections by page.
-> :pre-notice-ely-list-page {}